WIP: [Syntax widget] Create syntax doc for string delimiter (#352)
rescript-association/rescript-lang.org: misc_docs/syntax/language_string_delimiter.mdx

---

> id: "string-delimiter"

Reviewer: ```suggestion
id: "string-literal"
```

> keywords: ["string", "delimiter"]

Reviewer: ```suggestion
keywords: ["string"]
```

name: "\"\""

> summary: "This is the `string` delimiter."

Reviewer: ```suggestion
summary: "This is the `string` literal syntax."
```

category: "languageconstructs"
---

> The `string` delimiter is composed of two **double** quotes. Single quotes are reserved for the `char` type.

Reviewer: ```suggestion
A `string` literal is composed of two **double** quotes. Single quotes are reserved for the `char` type.
```
